dist/README

```diff
@@ -0,0 +1,26 @@
+The rkt binary and stage1 images provided in this tarball should be
+usable immediately after unpacking. rkt will use stage1-coreos.aci
+from this directory by default as a fallback. The steps below will
+make rkt more usable though (like making rkt available to everyone on
+the system, enabling rkt fetch for non-root users, integrating it with
```

Comment:
yes, thank you for the info. What I'm saying here is that maybe these instructions should include a pointer to something similar.